assets/src/dashboard/constants/index.js

```diff
@@ -64,22 +72,34 @@ export const ROUTE_TITLES = {
     'Template Details',
     'web-stories'
   ),
-  [APP_ROUTES.EDITOR_SETTINGS]: __('Editor Settings', 'web-stories'),
+  [APP_ROUTES.EDITOR_SETTINGS]: __('Settings', 'web-stories'),
   [APP_ROUTES.SUPPORT]: __('Support', 'web-stories'),
   DEFAULT: __('My Stories', 'web-stories'),
 };
 
 export const PRIMARY_PATHS = [
-  { value: APP_ROUTES.MY_STORIES, label: ROUTE_TITLES[APP_ROUTES.MY_STORIES] },
+  {
+    value: APP_ROUTES.MY_STORIES,
+    label: ROUTE_TITLES[APP_ROUTES.MY_STORIES],
+    Icon: HomeWithHeart,
+    IconActive: HomeWithHeartActive,
+  },
   {
     value: APP_ROUTES.SAVED_TEMPLATES,
     label: ROUTE_TITLES[APP_ROUTES.SAVED_TEMPLATES],
     inProgress: true,
+    Icon: FloppyDisk,
+    IconActive: FloppyDiskActive,
   },
   {
     value: APP_ROUTES.TEMPLATES_GALLERY,
     label: ROUTE_TITLES[APP_ROUTES.TEMPLATES_GALLERY],
+    Icon: Compass,
+    IconActive: CompassActive,
   },
+];
+
+export const SECONDARY_PATHS = [
```

Comment:
These are the links that are stuck to the bottom of the nav. I separated them out into their own constant.